### PR TITLE
fix redirecting to category search from other pages, e.g. profile

### PR DESCRIPTION
**Pull request by abid-salahi**

### CODE REVIEW

File: src/redux/constants/classifierTypes.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export const categories = [
-  { value: "", label: "" },
```
Comment (jenessatan) [inline src/redux/constants/classifierTypes.js]: Can't remove this. This array is used in all the forms. The first option is the default option of the dropdown menu and has to be an empty string to force users to choose from one of the actual categories

**Review by jenessatan — CHANGES_REQUESTED** [review]
Is there a bug that caused you to change this? This change would affect other parts of the application

File: src/redux/constants/classifierTypes.js (hunk 1 above)
Comment (abid-salahi) [reply src/redux/constants/classifierTypes.js]: oh shoot okay I wasnt aware

**Comment by abid-salahi** [comment]
The bug was basically when you redirected to a category from the drawer while you're on the /profile page. It goes to a wrong link and does something like http://localhost:3000/profile/items/category?... instead of http://localhost:3000/items/category?...

**Pull request merged**